docs/kratos/social-signin/30_apple.mdx

```diff
@@ -4,8 +4,60 @@ title: Add Apple as a social sign-in provider in Ory
 sidebar_label: Apple
 ---
 
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
```

Comment:
this should be an "mdx-code-block"

Reply:
Not sure I understand the problem with this. The actual warning is in an mdx-code-block (further down). This is just an import statement.

Reply:
Ok, now I understand. Apparently, the convention is that all non-pure Markdown gets wrapped in an mdx-code-block.